settings.gradle

```diff
@@ -28,11 +28,13 @@ include 'geode-pulse'
 include 'geode-assembly'
 include 'geode-rebalancer'
 include 'geode-lucene'
+include 'geode-lucene/geode-lucene-test'
```

Comment:
Can you explain more why we need a subproject just for the tests of lucene, instead of using the existing `test` source-set in the project?

Reply:
There are tests classes used in both test, integration and distributed test source sets and those sources sets cannot be dependent on each other.